bias:
- -2.108590939437387
- -15.636003746746526
- 0.800132302884682
- -0.2079094928554678
- 0.09305037146272349
- -0.12174466855632349
gripper_com_frame_id: ft_sensor0_wrench
gripper_com_pose:
- -0.0010148526443274158
- 0.001042110644991399
- 0.05319311978735169
- 0.0
- 0.0
- 0.0
gripper_mass: -0.9853498543483413
